profiles/common/modules/features/cce_basic_config/cce_basic_config.module

```diff
@@ -325,3 +325,32 @@ function cce_basic_config_tmgmt_dgt_html_filter_formats_containing_html() {
     'full_html',
   );
 }
+
+/**
+ * Implements hook_preprocess_container().
+ *
+ * It preprocesses the container theme in order to inject
+ * theme_hook_suggestions.
+ */
+function cce_basic_config_preprocess_container(&$variables) {
+  $element = $variables['element'];
+  $suggestion = 'container';
+
```

Comment:
ou should add an extra entry here, so you'll be able to use `container----wysiwyg.tpl.php` template only once.

```php
if (!empty($element['#bundle'])) {
  $variables['theme_hook_suggestions'][] = $suggestion . '____' . $element['#bundle'];
}
```